link-hint.el

```diff
@@ -418,6 +419,26 @@ Only search the range between just after the point and BOUND."
   :open-multiple t
   :copy #'kill-new)
 
+(defun link-hint--treemacs-open ()
```

Comment:
The name of most of the helpers for opening is something allong the lines of `link-hint--open-<type>`. Could you call this something like `link-hint--open-treemacs-button` or `link-hint--open-treemacs-entry`?